<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grid_dialog</class>
 <widget class="QDialog" name="grid_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Options for detailed specification of APBS grid parameters</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
      <widget class="QCheckBox" name="use_custom_checkBox">
       <property name="text">
        <string>Use Custom Grid</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QDialogButtonBox" name="dialog_buttons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Automatic Grid Generation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="verticalSpacing">
         <number>-1</number>
        </property>
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Method:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy">
             <cstring>auto_method_comboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="auto_method_comboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Memory Ceiling (MB):</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="buddy">
             <cstring>spinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="calculate_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Calculate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox">
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="singleStep">
             <number>128</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Grid Parameters</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="grid_tableWidget">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>65</number>
          </attribute>
          <row>
           <property name="text">
            <string>Coarse Grid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Fine Grid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Grid Center</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Grid Points</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>z</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
